music_assistant/common/models/enums.py

```diff
@@ -3,13 +3,25 @@
 from __future__ import annotations
 
 import contextlib
-from enum import StrEnum
-from typing import Self
+from enum import EnumType, StrEnum
 
-from music_assistant.common.helpers.util import classproperty
 
+class MediaTypeMeta(EnumType):
+    """Class properties for MediaType."""
 
-class MediaType(StrEnum):
+    @property
+    def ALL(self) -> list[MediaType]:  # noqa: N802
```

Comment:
`@classproperty` is a bit broken under mypy, e.g. https://github.com/python/mypy/issues/2563. Properties on the metaclass work like classproperty and have working typing. Maybe a constant of type `Final[tuple[MediaType, ...]]` would be better?

Reply:
yeah, this is only used as a shortcut on some places to reference all media types. feel free to adjust/modify